internal/operator/api/v1beta1/suite_test.go

```diff
@@ -0,0 +1,74 @@
+/*
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1beta1
+
+import (
+	"path/filepath"
+	"testing"
+
+	. "github.com/onsi/ginkgo"
```

Comment:
Note that `kubebuilder` autogenerates some test suite scaffolding for us too. I've never actually used `gingko` before, but it looks to be a more BDD style of testing like RSpec/Mocha/Jasmine